model/host/db.go

```diff
@@ -1663,6 +1663,30 @@ func FindExceedsSleepScheduleTimeout(ctx context.Context) ([]Host, error) {
 	return Find(ctx, q)
 }
 
+// ClearExpiredTemporaryExemptions clears all temporary exemptions from the
+// sleep schedule that have expired.
+func ClearExpiredTemporaryExemptions(ctx context.Context) error {
+	sleepScheduleTemporarilyExemptUntilKey := bsonutil.GetDottedKeyName(SleepScheduleKey, SleepScheduleTemporarilyExemptUntilKey)
+
+	res, err := evergreen.GetEnvironment().DB().Collection(Collection).UpdateMany(ctx, isSleepScheduleApplicable(bson.M{
+		sleepScheduleTemporarilyExemptUntilKey: bson.M{"$lte": time.Now()},
+	}), bson.M{
+		"$unset": bson.M{
+			sleepScheduleTemporarilyExemptUntilKey: 1,
+		},
+	})
+	if err != nil {
+		return err
+	}
+
+	grip.InfoWhen(res.ModifiedCount > 0, message.Fields{
```

Comment:
So this one is more like a debug log than for general monitoring, since clearing the temporary exemptions is part of expected behavior. It's only here as a way to detect when temporary exemptions expire, in case we need to trace/investigate issues with the sleep schedule machinery.

I'd like if we could get the modified document IDs too, but MongoDB doesn't expose an easy/efficient way to do that...